### PR TITLE
[no ticket] Clearer release steps in README

### PR DESCRIPTION
**Pull request by mlilynolting**

### CODE REVIEW

**Review by LizBaldo — APPROVED** [review]
Thank you!

File: README.md [hunk 1]
```diff
@@ -76,19 +76,19 @@ Run DB tests by projects will have fewer failures. Here's how you can run them b
...
 
-3. Get this terra-helm PR merged (you can merge it yourself) and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
+3. **Merge the automatically generated `terra-helm` PR.** You can merge this yourself. Another automatic commit will bump `leonardo`'s chart version. This will trigger another automatic commit
 in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile). Note that this commit will only auto-bump `dev`, and will be auto-merged.
 
-4. Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click the `SYNC APPS` button on the left upper corner. Select these boxes to sync:
+4. **Sync the deployed `leonardo` app in Argo.** Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click the `SYNC APPS` button on the left upper corner. Select these boxes to sync:
```
Comment (Qi77Qi) [inline README.md:89]: I don't think this is necessary...what happened this time is the bad commit was never merged in `terra-helmfile`

Comment (mlilynolting) [reply README.md:89]: These are just headers to make the steps in README more readable. Why not?

Comment (mlilynolting) [reply README.md:89]: Or do you mean that step 4 is not necessary and can be removed?

Comment (Qi77Qi) [reply README.md:89]: I meant there's no need to manually `Sync the deployed `leonardo` app in Argo.`

Comment (mlilynolting) [reply README.md]: Thanks!

**Review by Qi77Qi — APPROVED** [review]
thanks for the update

**Pull request merged**